.github/workflows/win-cuda-x64-build.yml

```diff
@@ -100,6 +95,12 @@ jobs:
         
         Get-ChildItem -Path $env:GITHUB_WORKSPACE\$env:binaryDir -Recurse
 
+    - name: Build the C# API and Run the C# Tests
+      run: |
+        $env:PATH = "${{ env.cuda_dir }}\\v${{ env.cuda_version }}\\bin;" + $env:PATH
```

Comment:
Do we need this? We have a `Add CUDA to PATH` item on line 72, would that not work?

Reply:
Yes this is needed. The env for each run is separated. The code in line 72 has no effect here.